lib/srv/desktop/windows_server.go

```diff
@@ -753,6 +755,20 @@ func (s *WindowsService) handleConnection(proxyConn *tls.Conn) {
 	}
 	desktop := desktops[0]
 
+	if desktop.NonAD() && buildType == "oss" {
```

Comment:
I think there's a constant you can use instead of the literal `"oss"`

Reply:
`modules.BuildOSS`? saw that was used above in `UpdateOSSDesktopAlert()`

Reply:
Yes, I forgot to change it here, fixed now.